File: task-03/Javascript/subtask-03.js
const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
  });
  
  function printStarPattern(n) {

    for (let i = 1; i <= n; i += 2) {
      let stars = '*'.repeat(i);
      let padding = ' '.repeat((n - i) / 2);
      console.log(padding + stars);
    }

    for (let i = n - 2; i >= 1; i -= 2) {
      let stars = '*'.repeat(i);
      let padding = ' '.repeat((n - i) / 2);
      console.log(padding + stars);
    }
  }
  
  readline.question('Enter a number: ', (num) => {
    let n = parseInt(num);
    printStarPattern(n);
    readline.close();
  });

File: task-03/Java/subtask03.java

    import java.util.Scanner;

public class subtask03 {
    public static void printStarPattern(int n) {
        for (int i = 1; i <= n; i += 2) {
            String padding = " ".repeat((n - i) / 2);
            String stars = "*".repeat(i);
            System.out.println(padding + stars);
        }

        for (int i = n - 2; i >= 1; i -= 2) {
            String padding = " ".repeat((n - i) / 2);
            String stars = "*".repeat(i);
            System.out.println(padding + stars);
        }
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a number: ");
        int n = scanner.nextInt();
        printStarPattern(n);
        scanner.close();
    }
}
